main/ertp/api/purse.md

```diff
@@ -1,78 +1,147 @@
 # Purse
-Purses hold an amount of digital assets of the same brand, but unlike payments, they are not meant to be sent to others. To transfer digital assets, a payment should be withdrawn from a purse. A purse's balance can rise and fall, through the action of deposit() and withdraw().
-
-The digital assets in purses and payments can be currency-like and goods-like digital assets, but they can also be other kinds of rights, such as the right to participate in a particular contract.
+`Purses` hold digital assets. They are created to hold assets of a single `brand`,
+and their `brand` cannot be changed. For example, you might create a `purse` to
+hold Quatloos. That `purse` can only hold Quatloos; it can't hold Moola or theater
+tickets or anything else. And you can't change that `purse` so that it now holds
+Moola instead; you need to create a `purse` that holds only Moola.
+
+Digital assets in `purses` and `payments` can be any of:
+- Currency-like, such as our imaginary Quatloos currency.
+- Goods-like digital assets, such as magic weapons for use in a game or theater tickets.
+- Other kinds of rights, such as the right to participate in a particular contract.
+
+While each `purse` can only hold assets of one `brand`, any number of `purses` can be
+created that hold that `brand`. So you could have, say, three Quatloos `purses`, your
+friend Alice could have eight Quatloos `purses`, and so on. 
+
+You change a `purse`'s balance by calling either `deposit()` (to add assets)
+or `withdraw()` (to remove assets) on it. A `purse` can be empty, which if it holds 
+a fungible currency means it has a value of 0. If it holds non-fungible theatre tickets, 
+it means it just doesn't have any tickets.
+
+Unlike `payments`, `purses` are not meant to be sent to others. 
+To transfer digital assets, you should withdraw a `payment` from a `purse` and
+send the `payment` to another party.
+
+## issuer.makeEmptyPurse()
+- Returns: `{Purse}`
+
+While not a method called on a `purse`, clearly it's important to know how
+to create a new `purse`. Call `makeEmptyPurse()` on the `issuer` associated
+with the `brand` of assets you want the new `purse` to hold. It returns the
+new `purse` with an empty balance. 
+```js
+const {issuer: quatloosIssuer} = makeIssuerKit('quatloos');
+quatloosPurse1 = quatloosIssuer.makeEmptyPurse();
+quatloosPurse2 = quatloosIssuer.makeEmptyPurse();
+```
 
 ## purse.getCurrentAmount()
 - Returns: `{Amount}`
 
-Get the current `amount` contained in this purse, confirmed by the
-issuer. Note that this amount will change from call to call if assets
-have been deposited or withdrawn between calls. 
+Get an `amount` describing the current digital assets balance in the `purse`.
+This `amount` is confirmed by the `issuer` for the `brand` the `purse` can hold.
```

Comment:
```suggestion
```

Reply:
This doesn't make sense. I think if this came from the types files, we have an issue on agoric-sdk to remove it.